source/App/source/Common.Abstractions/Common.Abstractions.csproj

```
@@ -23,7 +23,7 @@ limitations under the License.
 
   <PropertyGroup>
     <PackageId>Energinet.DataHub.Core.App.Common.Abstractions</PackageId>
-    <PackageVersion>14.0.2$(VersionSuffix)</PackageVersion>
+    <PackageVersion>14.0.32$(VersionSuffix)</PackageVersion>
```

```suggestion
    <PackageVersion>14.0.3$(VersionSuffix)</PackageVersion>
```

Looks like a copy/paste typo